### PR TITLE
docs(aio): Upgrade server-side rendering example to Angular V6

### PR DESCRIPTION
**Pull request by brandonroberts**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[x] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

Once #23521 lands, I'll rebase this branch

### CODE REVIEW

**Comment by jenniferfell** [comment]
@vikerman Hi. Igor asked me to reach out to you for review and approval on this. Hoping to merge today. It's a change to universal examples to bring them current for v6. Thanks!

**Comment by vikerman** [comment]
Looking into this now...

File: aio/content/examples/universal/server.ts [hunk 1]
```diff
@@ -51,7 +51,7 @@ app.get('*.*', express.static(join(DIST_FOLDER, 'browser')));
 // #docregion navigation-request
 // All regular routes use the Universal engine
 app.get('*', (req, res) => {
-  res.render(join(DIST_FOLDER, 'browser', 'index.html'), { req });
```
[the remark this answers is not in the document]
Comment (brandonroberts) [reply aio/content/examples/universal/server.ts]: The change was based off the universal-start upgrade to V6. Did fix the change from `index.html` to `index`

**Review by vikerman — APPROVED** [review]
Approving. Please ensure sample works with the changes.

**Pull request closed without merging**